worldcup/README.md

```diff
@@ -22,21 +22,111 @@ Mexico: 0.1% chance of winning
 
 ## Background
 
-TODO
+* In soccer, teams are given [FIFA Ratings](https://en.wikipedia.org/wiki/FIFA_World_Rankings#Current_calculation_method), which are numerical values representing the team's relative skill levels. Higher FIFA ratings indicate better previous game results, and we can determine the probability that a team wins a matchup if we have both teams' FIFA ratings.
+
+* In the World Cup, 16 teams begin in the knockout round. At each round, half the teams are eliminated. When two teams remain, the winner of the final match is the champion and the loser is the runner-up. 
+
+* The FIFA Ratings from just before the two previous World Cups are here: [May 2018 Men's FIFA Ratings](https://www.fifa.com/fifa-world-ranking/ranking-table/men/rank/id12189/) and [March 2019 Women's FIFA Ratings](https://www.fifa.com/fifa-world-ranking/ranking-table/women/rank/ranking_20190329/)
+  
 
 ## Implementation Details
 
-TODO
+* Complete the implementation of `volume.c` at right, such that it simulates a number of tournaments and outputs each team's probability of winning, for all probabilities greater than `0%`.
+
+* In `main()`, your program should read in the file indicated by the user into the list `bracket`. This list should be a list of dictionaries, each dictionary containing `team` and `rating` keys, where the `rating` is a numerical value. 
+
+  * Two files are provided: `2018m.csv` and `2019w.csv`. 
+
+  * In each, the bracket of teams, in order, is provided along with their FIFA Ratings. 
+    
+    * Specifically, in the first round, the team in line 1 will play the team in line 2. The team in line 3 will play the team in line 4. Since there are 16 total lines, the first round will contain 8 total games. 
+
+    * In the second round, the winner between lines 1 and 2 will play the winner between lines 3 and 4, etc.
+
+    * The first few lines in `2018m.csv` look like this: 
+    
+    ```
+      team,rating
+      Uruguay,976
+      Portugal,1306
+      France,1166
+    ```
+
+* After, in `main()`, your program should simulate `N` tournaments, which, notice, has already been defined as `1000`. For each tournament, your program should return the winner and keep track of the number of wins for each team in the dictionary `counts`.
+
+* In `simulate_tournament(bracket)`, your program should return the name of the winning team of one simulation of the given bracket. Your program should iterate through as many rounds as necessary to output one single winner of the tournament. 
+
+* In `simulate_round(bracket)`, your program should simulate a single round of the tournament and return the winners of each game in that round.  
+  
+  * Note that in the first round, the first team should play the second, the third team should play the fourth, and so on. 
+
+  * Additionally, for example, if the bracket given has 4 teams (third round), your program should simulate 2 games and return a list of the 2 winning teams. 
```

Comment:
I think we need to clarify something about the results being "in order" (that is, the winner between team 0 and team 1 should appear before the winner between team 2 and team 3 in the resulting list)